capstone-ui/src/components/App/App.jsx

```diff
@@ -121,6 +157,53 @@ function App() {
         }
       });
   }
+
+  const saveInterests = () => {
+    setIsLoading(true);
+    axios
+      .post(`${config.API_BASE_URL}/profile/interests`, {
+        interests: {
+          skill: selectedSkill
+            ? selectedSkill.value
+            : newSkill
+            ? newSkill
+            : null,
```

Comment:
Nit: Since it is better to avoid double nesting, try this:
```
skill: selectedSkill 
  ? selectedSkill.value : newSkill ?? null
```

The  ?? is called "Nullish coalescing" operator. I'm not sure if your build supports it, but try it out.